scripts/measures/2018/enrich-measures-data.js

```diff
@@ -0,0 +1,164 @@
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+
+const currentYear = 2018;
+const piRelations = require('../../../util/measures/' + currentYear + '/pi-measure-relations.json');
+const cpcPlusGroups = require('../../../util/measures/' + currentYear + '/cpc+-measure-groups.json');
+const stratifications = require('../../../util/measures/2017/additional-stratifications.json');
+
+const QUALITY_CATEGORY = 'quality';
+const measuresDataPath = process.argv[2];
+const outputPath = process.argv[3];
+const qpp = fs.readFileSync(path.join(__dirname, measuresDataPath), 'utf8');
+fs.writeFileSync(path.join(__dirname, outputPath), enrichMeasures(JSON.parse(qpp)));
+
+function enrichMeasures(measures) {
+  enrichPIMeasures(measures);
+  enrichCPCPlusMeasures(measures);
+  enrichInverseMeasures(measures);
+  enrichStratifications(measures);
+  mergeGeneratedEcqmData(measures);
+  addRequiredRegistrySubmissionMethod(measures);
+  enrichClaimsRelatedMeasures(measures);
+  return JSON.stringify(measures, null, 2);
+};
+
+/**
+ * Will add extra metadata to PI measure that are not directly available
+ * in machine inferable format at https://qpp.cms.gov/api/v1/aci_measures
+ * After this function executes, an ACI measure will have reporting category and substitutes.
```

Comment:
could you update ACI here and in other places in this pr to read PI?

Reply:
Will do, I thought I got all the ACI references but I missed a some. Good catch.